deploy/api.yaml

```diff
@@ -0,0 +1,157 @@
+        passthroughBehavior: "when_no_match"
+  /question:
+    get:
+      parameters:
+        - $ref: "#/components/parameters/SessionHeader"
+      responses:
+        "200":
+          description: "200 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Question"
+        "201":
+          description: "201 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Question"
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVQuestionFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+  /answer:
+    post:
+      parameters:
+        - $ref: "#/components/parameters/SessionHeader"
+      responses:
+        "201":
+          description: "201 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Answer"
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVAnswerFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+components:
+  parameters:
+    SessionHeader:
+      name: session-id
+      in: header
+      description: A UUID generated by the KBV API to act as a primary key for the KBVSessionTable in DynamoDB
+      required: true
+      schema:
+        type: string
+  schemas:
+    Session:
+      type: "object"
+      properties:
+        session-id:
+          type: "string"
+    Error:
+      title: "Error Schema"
+      type: "object"
+      properties:
+        message:
+          type: "string"
+    Question:
+      type: "object"
+      properties:
+        questionID:
+          type: "string"
+        text:
+          type: "string"
+        tooltip:
+          type: "string"
+        answerHeldFlag:
```

Comment:
What needs this property?

Reply:
The property is sent back in the `Question` payload we receive from the Experian API. It is currently part of the payload and I have not really dug deep into what is its exact need. We can ask the API to ignore it but we may need it in perhaps in the future flows.